### PR TITLE
Remove unscoped find in candidate controllers

### PR DESCRIPTION
**Pull request by tijmenb**
## Context

cc @stevehook 

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/controllers/candidate_interface/degrees/degrees_base_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module CandidateInterface
+  module Degrees
+    class DegreesBaseController < CandidateInterfaceController
```
Comment (malcolmbaig) [inline app/controllers/candidate_interface/degrees/degrees_base_controller.rb:3]: Namespaced, so can just call it `BaseController`. Will then be consistent with other example in the codebase.

File: app/controllers/candidate_interface/other_qualifications/other_qualifications_base_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module CandidateInterface
+  class OtherQualifications::OtherQualificationsBaseController < CandidateInterfaceController
```
Comment (malcolmbaig) [inline app/controllers/candidate_interface/other_qualifications/other_qualifications_base_controller.rb:2]: As above, can just call this `OtherQualifications::BaseController`

**Review by malcolmbaig — COMMENTED** [review]
LGTM, just a couple comments

**Review by stevehook — APPROVED** [review]
Tested review app, can edit my own qualifications but get 404 when I mangle the URL now :+1:

Code looks good, agree with @malcolmbaig 's suggestions.

**Review by tvararu — APPROVED** [review]
Agree with Malcolm's comment :+1:

**Review by malcolmbaig — APPROVED** [review]

**Review by malcolmbaig — APPROVED** [review]

**Pull request merged**